src/main/java/com/rackspace/ceres/app/config/AppProperties.java

```diff
@@ -110,4 +110,8 @@
   @NotNull Integer tagValueLimit = 50;
 
   List<String> excludedTagKeys;
+
+  String ingestStartTime;
```

Comment:
Take this as Duration object instead of String

Reply:
Fixed it